cli/viam/main.go

```diff
@@ -229,16 +236,14 @@ func main() {
 				},
 			},
 			{
-				Name:  "data",
-				Usage: "work with data",
+				Name:            "data",
+				Usage:           "work with data",
+				HideHelpCommand: true,
 				Subcommands: []*cli.Command{
 					{
-						Name:  "export",
-						Usage: "download data from Viam cloud",
-						UsageText: fmt.Sprintf("viam data export <%s> <%s> [%s] [%s] [%s] [%s] [%s] [%s] [%s] [%s] [%s] [%s] [%s] [%s] [%s] [%s]",
```

Comment:
`data export` and `data delete` had _super_ long usage texts that were pretty overwhelming and felt redundant with the options help messages below. I think all that's important is that users know which options are available and which are required.